lib/is-semantic-message.js

```diff
@@ -1,11 +1,10 @@
 const commitTypes = Object.keys(require('conventional-commit-types').types)
-const parseCommitMessage = require('parse-commit-message').parse
+const { validate } = require('parse-commit-message')
 
 module.exports = function isSemanticMessage (message) {
-  try {
-    const { type } = parseCommitMessage(message)
-    return commitTypes.includes(type)
-  } catch (e) {
-    return false
-  }
+  const { error, value: commits } = validate(message, true)
+  if (error) return false
```

Comment:
How about `console.error(error)` here before returning, instead of swallowing the error.

Reply:
Ookey, no problem.

Reply:
Done.